kafka-client/src/main/java/dev/responsive/kafka/internal/stores/PartitionedOperations.java

```diff
@@ -117,6 +115,29 @@ public static PartitionedOperations create(
     );
   }
 
+  private static RemoteKVTable createCassandra(
```

Comment:
I won't push back on it if you just want to get something merged, but I'm not a fan of this structure or these methods being here/the way they're named.

Seems like we should have a client interface, perhaps with just these methods: `createKVTable`, and `createWindowTable` (and eventually I guess `createSessionTable`. The client interface could also have a `clientType` method that returns `CASSANDRA` or `MONGO_DB` and let us remove the enum entirely (assuming we also replace the config value with plain strings)

Reply:
definitely agree that the current state of things isn't in tip top shape from abstractions, this was what I could throw together with the fewest lines of code changed while still improving the situation.